web/wiki/src/pages/Category/index.ts

```diff
@@ -0,0 +1,3 @@
+import { Category } from './Category'
+
+export { Category }
```

Comment:
You also could do `export { Category } from './Category` just in one line.

Reply:
Done!